### ISSUE TITLE
Non-Code Contributions

### ISSUE THREAD

**Issue by WillAyd** [innobi/pantab#366]
For anyone coming here as part of Hacktoberfest, pantab would gladly accept any non-code contributions that are outlined in this article:

https://hacktoberfest.com/participation/

Feel free to drop a line here and we can discuss!

**Comment by Ganesh2409** [comment]
can i know more about it !

**Comment by WillAyd** [comment]
Sure! This issue is a little more open ended so depends on what skills / interests each contributor may have. Was there anything from the hacktoberfest link that you were interested in?

**Comment by Ganesh2409** [comment]
yeah I am interested in these topics --> Talks or presentations , Technical Documentation , Writing ,Technical blog post or tutorial
